Shared/Samples/Style point with distance composite scene symbol/StylePointWithDistanceCompositeSceneSymbolView.swift

```diff
@@ -0,0 +1,139 @@
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The distance from the target object to the camera in meters.
+    @State private var cameraDistance = Measurement(value: 0, unit: UnitLength.meters)
+    
+    var body: some View {
+        SceneView(
+            scene: model.scene,
+            cameraController: model.cameraController,
+            graphicsOverlays: [model.graphicsOverlay]
+        )
+        .overlay(alignment: .top) {
+            VStack {
+                Text("Zoom in and out to see the symbol change.")
+                    .frame(maxWidth: .infinity)
+                    .padding(8)
+                    .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+                
+                HStack {
+                    Spacer()
+                    HStack {
+                        Text("Distance:")
+                        Text(cameraDistance, format: .measurement(
+                            width: .narrow,
+                            usage: .asProvided,
+                            numberFormatStyle: .number.precision(.fractionLength(0))
+                        ))
+                    }
+                    .padding()
+                    .background(.ultraThinMaterial)
+                    .cornerRadius(10)
+                    .shadow(radius: 3)
+                }
+                .padding(.trailing, 8)
+            }
+        }
+        .task {
+            for await newDistance in model.cameraController.$cameraDistance {
+                cameraDistance.value = newDistance
+            }
+        }
+    }
+}
+
+private extension StylePointWithDistanceCompositeSceneSymbolView {
+    /// The view model for the sample.
+    class Model: ObservableObject {
+        /// A scene with an imagery basemap and world elevation surface.
+        let scene: ArcGIS.Scene = {
+            let scene = Scene(basemapStyle: .arcGISImagery)
+            let elevationSource = ArcGISTiledElevationSource(url: .worldElevationService)
+            scene.baseSurface.addElevationSource(elevationSource)
+            return scene
+        }()
+        
+        /// The camera controller focused on the plane graphic.
+        private(set) var cameraController: OrbitGeoElementCameraController!
+        
+        /// The graphics overlay for the plane graphic.
+        let graphicsOverlay: GraphicsOverlay = {
+            let graphicsOverlay = GraphicsOverlay()
+            graphicsOverlay.sceneProperties.surfacePlacement = .relative
+            return graphicsOverlay
+        }()
+        
+        /// The plane graphic created from a distance composite symbol.
+        private let planeGraphic: Graphic = {
+            // Create the different symbols.
+            let planeSymbol = ModelSceneSymbol(url: .bristol, scale: 100)
+            let coneSymbol = SimpleMarkerSceneSymbol.cone(color: .red, diameter: 200, height: 600)
```

Comment:
Should this use a centered anchor? The cone seems to be positioned a bit higher than the plane and the jump is noticeable.

```suggestion
            let coneSymbol = SimpleMarkerSceneSymbol.cone(color: .red, diameter: 200, height: 600, anchorPosition: .center)
```